eternalcore-paper/src/main/java/com/eteranlcode/containers/AdditionalContainerPaper.java

```diff
@@ -0,0 +1,33 @@
+package com.eteranlcode.containers;
+
+import io.papermc.lib.PaperLib;
+import io.papermc.lib.environments.Environment;
+import org.bukkit.ChatColor;
+import org.bukkit.entity.Player;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.logging.Logger;
+
+public final class AdditionalContainerPaper {
+
+    private static final Environment environment = PaperLib.getEnvironment();
+
+    public static void openAdditionalContainer(Player player, Logger logger, @NotNull AdditionalContainerType type) {
+
+        if (!environment.isPaper()) {
+            player.sendMessage(ChatColor.RED + "Anvil command feature is not supported on this server. Please contact the server administrator and check console!");
```

Comment:
Czy to jest dobry pomysł? Ludzie mogą to wykorzystać, spamować tą komendą aby wywołać spam w konsoli